### PR TITLE
Create user confirmation message

### PR DESCRIPTION
**Pull request by VishalThapaliya**
Display success message when create or update a user.

Covers [BR-583](https://bonitasoft.atlassian.net/browse/BR-583)

### CODE REVIEW

File: uid-pages/page-admin-user-list/src/adminUserListBonitaV1.json [hunk 1]
```diff
@@ -3075,19 +3187,26 @@
...
+        "    $data.createUserData.hasRecentlyBeenCreated = true;",
+        "}",
+        "",
+        "// After the status is changed",
+        "if ($data.changeStatusData.statusCode !== '' && $data.changeStatusData.hasRecentlyBeenActivated === false && $data.changeStatusData.hasRecentlyBeenDeactivated === false) {",
         "    $data.userListData.timestamp = new Date().getTime();",
-        "    $data.createUserData.statusCode = '';",
-        "    $data.changeStatusData.statusCode = '';",
+        "    $data.changeStatusData.hasRecentlyBeenActivated = true;",
+        "    $data.changeStatusData.hasRecentlyBeenDeactivated === true;",
```
Comment (DumitruCorini) [inline uid-pages/page-admin-user-list/src/adminUserListBonitaV1.json:3209]: This seems weird, why are we verifying that hasRecentlyBeenDeactivated is equal to true?

**Review by DumitruCorini — CHANGES_REQUESTED** [review]
Also, why do you have both hasRecentlyBeenActivated and hasRecentlyBeenDeactivated? Both are used in the same way, could we not just keep one and it would work all the same?

**Review by abirembaut — APPROVED** [review]
LGTM

File: uid-pages/page-admin-user-list/src/adminUserListBonitaV1.json (hunk 1 above)
Comment (DumitruCorini) [reply uid-pages/page-admin-user-list/src/adminUserListBonitaV1.json]: Done

**Review by DumitruCorini — APPROVED** [review]

**Pull request merged**